controllers/controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectCategories } = require('../models/model.js')
+
+exports.getCategories = (req,res,next) => {
+    return selectCategories().then(categories => {
+        console.log(categories)
```

Comment:
Please remove all console.logs before submitting PR in future - shouldn't be in "production" code. Apart from the one in the 500s error handler.